src/content/docs/en/guides/styling.mdx

```diff
@@ -520,11 +520,11 @@ Svelte in Astro also works exactly as expected: [Svelte Styling Docs][svelte-sty
 
 ## Markdown Styling
 
-Any Astro styling methods are available to a [Markdown layout component](/en/core-concepts/layouts/#markdownmdx-layouts).
+Any Astro styling methods are available to a [Markdown layout component](/en/core-concepts/layouts/#markdownmdx-layouts). Please note that layout styling is not the same as the rendered Markdown content styling.
 
-You can apply global styles to your Markdown content by adding [`<style>` tags](#global-styles) and [imported stylesheets](#external-styles) to the layout that wraps your page content. You can also add [CSS integrations](#css-integrations) including [Tailwind](/en/guides/integrations-guide/tailwind/).
+You can apply global styles to your Markdown content by adding [`<style>` tags](#global-styles) and/or [imported stylesheets](#external-styles) to the layout that wraps your page content. Imported stylesheets are the simplest way to only apply stylings to rendered page content, however are still subject to [Astro's cascading order](#cascading-order).
```

Comment:
```suggestion
You can apply global styles to your Markdown content by adding [imported stylesheets](#external-styles) to the layout that wraps your page content. It is also possible to style your Markdown with [`<style is:global>` tags](#global-styles) in the layout component.  Note that any styles added are subject to [Astro's cascading order](#cascading-order), and you should check your rendered page carefully to ensure your styles are being applied as intended.
```
We don't describe methods as "simple" or "easy" in docs. But if it represents a happier path, we can put it first, as its own statement, and make the global style tags a bit of an afterthought. I've also added the global attribute here match the wording of "global styles."